src/components/formik-radiobutton/FormikRadiobutton.tsx

```diff
@@ -98,6 +99,25 @@ const ErrorMessageStyled = styled.span`
   margin-bottom: var(--spacing-xs);
 `;
 
+const TooltipIconStyled = styled.span`
+  font-size: var(--icon-size-xs);
+  line-height: 0;
+  margin-right: var(--spacing-s);
+`;
+
+const TooltipWrapperStyled = styled.div`
+  margin-right: var(--spacing-s);
+`;
+
+const WrapperDivStyled = styled.div`
```

Comment:
Naam veranderen naar WrapperStyled (Geen html elementen in de namen voor styledcomponents) of zo iets.